packages/tpetra/core/ext/TpetraExt_MatrixMatrix_SYCL.hpp

```diff
@@ -0,0 +1,826 @@
+// @HEADER
```

Comment:
Sigh.  Yet another copy of this MatrixMatrix code.  @trilinos/tpetra has to fix this duplication.  How does this SYCL version differ from other versions?  We'd like to capture this info so that we can someday merge all these duplicate files.  Thanks.

Note, I am assuming that this file is merely a copy of other MatrixMatrix files; I have not reviewed the code.  If there are significant differences, one of @trilinos/tpetra 's MatrixMatrix experts should review.

Reply:
Yes, this is a copy of the other `MatrixMatrix` files replacing the execution/memory space with the ones corresponding to `SYCL`.
I would think that it's not too difficult to avoid the code duplication for these files but that's out-of-scope of this pull request IMHO.

Reply:
I agree; this duplication should be handled by @trilinos/tpetra